src/refactor_nrepl/ns/pprint.clj

```diff
@@ -35,11 +35,19 @@
                             (printf "%s " libspec))))))
                   ordered-libspecs))))
 
-(defn insert-clause-delimiter []
+(defn- insert-clause-delimiter []
   (if (:insert-newline-after-require *config*)
     (println)
     (print " ")))
 
+(defn- pprint-libspec [libspec]
+  ;; If a vector gets too long `pprint` will print one element per line.
+  ;; This puts `:as` and `:refer` on their own line, which causes the ns form
+  ;; to take up too much vertical space.
+  (printf (str/replace (with-out-str (pprint libspec))
+                       (re-pattern (str "(:as|refer)" (System/lineSeparator)))
```

Comment:
`(re-pattern (str "(:as|refer)" (System/lineSeparator))` will be invoked once per `pprint-libspec` call, can be optimized

Reply:
It's less efficient for sure, but the code is much easier to work with this way.

Since `n` is very low (nobody has hundreds of libspecs) the performance difference will be very small.  I'd rather leave this as is and prioritize readability/maintainability.

Reply:
Tbh this doesn't like a huge compromise wrt maintainability:

```clj
(def clause-re (re-pattern (str "(:as|refer)" (System/lineSeparator))
```

> Since n is very low (nobody has hundreds of libspecs)

I don't know, think of doing a project-wide refactor over 1000 files, each having 10 libspecs as a conservative average.

Saving 1s or such can be nice.

Reply:
Think I misunderstood a little; thanks for elaborating.  You're right of course 🙂 

Fixed.